x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -366,6 +405,25 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
             );
           }}
         />
+        {currentColumn.params.orderBy.type === 'rare' && (
+          <ValuesInput
+            value={currentColumn.params.orderBy.maxDocCount}
+            label={i18n.translate('xpack.lens.indexPattern.terms.maxDocCount', {
+              defaultMessage: 'Max doc count per term',
+            })}
+            maxValue={100}
```

Comment:
Where this value come from?

Reply:
Moved it into a constant, magic numbers are bad